presto-main/src/main/java/com/facebook/presto/server/security/OpenLdapFilter.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.server.security;
+
+import com.google.inject.Inject;
+
+import javax.naming.NamingException;
+
+import static java.util.Objects.requireNonNull;
+
+public class OpenLdapFilter
+        extends LdapFilter
+{
+        private final String ldapBaseDn;
+
+        @Inject
+        public OpenLdapFilter(LdapServerConfig config)
+        {
+                super(config);
+                ldapBaseDn = requireNonNull(config.getBaseDn(), "ldap.base.dn is null");
```

Comment:
In general FB doesn't abbreviate names so if there are instances other than this variable name, they should also be expanded. In this case `ldapBaseDistinguishedName` might be too verbose as Andrii points out so maybe just `distinguishedName`?